### ISSUE TITLE
Sequence of actions changes after rule is created

### ISSUE THREAD

**Issue by AkubaD** [HRHR-project/e-registry-app#92]
I first observed this in February, so it is not related to a recent deploy. 
When you create a rule and have a sequence of actions, after you save the rule and reopen it, the sequence of the actions changes.
See changed sequence below. I have been able to replicate it twice consistently when I create a rule with multiple actions

![image](https://user-images.githubusercontent.com/36447512/39818261-c15092a0-53a0-11e8-9f07-32b0d5f5c334.png)

And there is no way to reorganize actions - unless you delete them and recreate them. And when you do that, the system changes the sequence. Here, I created my 1st action again called '1st Action anew'. It is the last action in the list
![image](https://user-images.githubusercontent.com/36447512/39818611-9ad2cfde-53a1-11e8-9da3-6909792b0004.png)

But once I saved the rule, the action has now moved to second place
![image](https://user-images.githubusercontent.com/36447512/39818747-e5d07928-53a1-11e8-9d43-ef273c3c8a79.png)

**Comment by lukasewvd** [comment]
This is not related to the E-Registry App, please create an issue for the Maintenance here: https://jira.dhis2.org.

**Comment by AkubaD** [comment]
@lukasewvd 
I tried to replicate this in 2.29 demo (https://play.dhis2.org)
 I created a rule with multiple actions, saved it and added one more action later. The sequence stayed the same - meaning it works ok, unlike in our instance. I rechecked in our instance today, and it is still an issue.
I'm not sure about reporting on Jira an issue that is not replicable for them...
This is the rule that I used to test in our instance https://bd-eregistry.dhis2.org/dhis/dhis-web-maintenance/#/edit/programSection/programRule/NTWqEs9e8G9
And this is the rule in the DHIS2 demo https://play.dhis2.org/2.29/dhis-web-maintenance/#/edit/programSection/programRule/kUmMxGe0yF1
